### DISCUSSION TITLE
Comet ML Logger with `ddp`

### DISCUSSION THREAD

**Discussion by FrancescoSaverioZuppichini** [Lightning-AI/pytorch-lightning#8683] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi Guys,

Is there a way to correctly log with comet with the `ddp` accelerator? I am not able to average the loss between GPUs

Thanks,

Francesco

**Comment by tchaton** [comment]
Dear @FrancescoSaverioZuppichini,

The loss should be atomically reduced on epoch end across batches and processes. Mind sharing a snippet of code ?

Best,
T.C
